twitter_account.rb

```diff
@@ -1,10 +1,14 @@
-require 'active_record'
 require 'mysql2'
 
-ActiveRecord::Base.establish_connection(
-    "adapter" =>"mysql2",
-    "database" => "twitter_analytics_acquirer"
-)
+class TwitterAccount
+  @client = Mysql2::Client.new(host: "localhost", username: "root", password: '', database: 'twitter_analytics_acquirer')
 
-class TwitterAccount < ActiveRecord::Base
+  def self.all
+    @client.query("SELECT `twitter_accounts`.* FROM `twitter_accounts`").each
+  end
```

Comment:
It may be cooler to select needed info.
But I think this method is easy to understand, so I choose this.